includes/integrations/wordpress/class-echodash-wordpress.php

```
@@ -117,6 +226,461 @@
+	}
+
+	/**
+	 * Handle new blog post comment.
+	 *
+	 * @since 2.0.0
+	 *
+	 * @param int        $comment_id     The comment ID.
+	 * @param int|string $comment_approved 1 if approved, 0 if not, 'spam' if spam.
+	 * @param array      $commentdata    Comment data.
+	 */
+	public function comment_post( $comment_id, $comment_approved, $commentdata ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$commentdata'. (Unused Code Rules)

(UnusedFormalParameter)